README.md

```diff
@@ -61,23 +61,23 @@ export default PageLayout;
+| **render**           | React.ReactNode      | **Yes**  | Unfortunately `react-helmet` and `next/head` are strict with how they accept meta tags. `react-helmet` doesn't support nesting. Whereas Next.JS only supports some children and not all, therefore a render function is required.                                                                                      |
+| **pageTitle**        | string               | **No**   | Unique page title that describes the page, such as `Home`, `About` etc. etc.                                                                                                                                                                                                                                           |
+| **siteTitle**        | string               | **Yes**  | Title of the site, usually the organization / brand name. If `pageTitle` and `siteTitle` are the same, only this shows.                                                                                                                                                                                                |
+| **titleTemplate**    | string               | **No**   | Title template used to display `pageTitle` and `siteTitle` in a template, displays the values using corresponding `[pageTitle]` and `[siteTitle]`. Example template: "[pageTitle] &#124; [siteTitle]".                                                                                                                 |
+| **description**      | string               | **Yes**  | A one to two sentence description of your webpage. Keep it within 160 characters, and write it to catch the user's attention.                                                                                                                                                                                          |
+| **baseSiteUrl**      | string               | **Yes**  | Base site URL, excluding trailing slash.                                                                                                                                                                                                                                                                               |
+| **pagePath**         | string               | **No**   | The path of the current page, excluding leading slash.                                                                                                                                                                                                                                                                 |
+| **disableCanonical** | string               | **No**   | Disable canonical if not desired, defaults to to `false`.                                                                                                                                                                                                                                                              |
```

Comment:
This is wrong. It's a boolean not a string. You can apply my suggestion without having to re-commit btw
```suggestion
| **disableCanonical** | boolean               | **No**   | Disable canonical if not desired, defaults to to `false`.                                                                                                                                                                                                                                                              |
```